pkg/kubelet/server/server.go

```diff
@@ -798,6 +804,66 @@ func (s *Server) getPortForward(request *restful.Request, response *restful.Resp
 	proxyStream(response.ResponseWriter, request.Request, url)
 }
 
+func trimPath(path string) string {
+	parts := strings.SplitN(strings.TrimPrefix(path, "/"), "/", 3)
+	if len(parts) == 0 {
+		return path
+	}
+
+	if parts[0] != "metrics" || len(parts) == 1 {
+		return parts[0]
+	}
+
+	metricsPaths := []string{"cadvisor", "resource", "probes"}
+
+	for _, p := range metricsPaths {
+		if parts[1] == p {
+			return fmt.Sprintf("%s/%s", parts[0], p)
+		}
+	}
+	return parts[0]
+}
+
+func isLongRunningRequest(path string) bool {
+	longRunningRequestPaths := []string{"exec", "attach", "portforward", "debug"}
```

Comment:
we might want to list these in the help-text of the long running metric

Reply:
Good point! Will fix it!